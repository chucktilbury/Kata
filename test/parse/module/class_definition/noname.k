;-------------------------------------------------------------------------------
; Verify that anonymous classes are syntax error
;-------------------------------------------------------------------------------

class {}
