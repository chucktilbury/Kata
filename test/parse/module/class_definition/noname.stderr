Syntax: /home/chuck/Src/Kata/test/parse/module/class_definition/noname.k: 5: 8: expected a SYMBOL but got {
Syntax: /home/chuck/Src/Kata/test/parse/module/class_definition/noname.k: 5: 8: expected module item or end of file but got class
Syntax: /home/chuck/Src/Kata/test/parse/module/class_definition/noname.k: 5: 8: module cannot be empty
